cnf-tests/docgen/e2e.json

```
@@ -1,6 +1,7 @@
 {
     "[bondcni] bond over macvlan should be able to create pod with bond interface over macvlan interfaces": "Verifies bond interface on top of macvlan interfaces is created correctly",
     "[dpdk] Downward API Volume is readable in container": "Verifies that the container downward API volumes are readable and match the Pod inforamtion details",
+    "[dpdk] Rootless vhostnet Validate traffic on a rootless pod should receive packets on the tap and vlan subinterfaces": "XXXXXXXX",
```

Can you change this to
```suggestion
    "[dpdk] Rootless vhostnet Validate traffic on a rootless pod should receive packets on the tap and vlan subinterfaces": "Verifies a rootless pod can receive packets on a tap and vlan subinterface",
```

and run
```
make check-tests-nodesc validate-test-list
```

that should update `TESTLIST.md`

---

thanks

---

done